ui/ui-components/components/source/FormMappingSelector.tsx

```diff
@@ -138,46 +140,48 @@ const FormMappingSelector: React.FC<Props> = ({
         </FormInputContainer>
         {columnExample && renderExamples(columnExample)}
       </Col>
-      <Col>
-        {hasAvailableProperties && (
-          <>
-            <FormInputContainer
-              controlId="mapping_property"
-              label="Mapped to Grouparoo Property"
-              required
-            >
-              <Form.Control
-                as="select"
+      {!mappingDisabled && (
+        <Col>
+          {hasAvailableProperties && (
```

Comment:
Consolidate these two checks into one. I think where mappingDisabled is located (above the Col) is the right place to check.

Note that in the first `<Col>` above you will also need to apply similar logic for the medium sizing.